app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -198,22 +195,24 @@ class LoggedInActivity : BaseActivity(R.layout.activity_logged_in) {
             }
         }
 
-        if (shouldShowTooltip(getLastOpen())) {
+        val currentEpochDay = LocalDate.now().toEpochDay()
```

Comment:
`java.time` has a class called `Instant` which represents epoch-time. I can recommend that you use that instead: `Instant.now()`

Reply:
Nevermind! I'm wrong